plugins/swiot/src/config/swiotconfig.cpp

```diff
@@ -201,10 +177,11 @@ void SwiotConfig::externalPowerSupply(bool ps)
 	}
 }
 
-void SwiotConfig::initTutorialProperties()
+void SwiotConfig::modeAttributeChanged(std::string mode)
```

Comment:
nitpick, but can this be a QString?

Reply:
I believe it's currently unnecessary since this modification will impact a larger number of classes.